B/Blosc2/bundled/patches/blosc_armv7l.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/blosc/CMakeLists.txt b/blosc/CMakeLists.txt
+index 6fdf399..412d475 100644
+--- a/blosc/CMakeLists.txt
++++ b/blosc/CMakeLists.txt
+@@ -194,7 +194,7 @@ if(COMPILER_SUPPORT_NEON)
+         # Only armv7l needs special -mfpu=neon flag; aarch64 doesn't.
+       set_source_files_properties(
+             shuffle-neon.c bitshuffle-neon.c
+-            PROPERTIES COMPILE_FLAGS "-mfpu=neon")
++            PROPERTIES COMPILE_FLAGS "-mfpu=neon -flax-vector-conversions")
+     endif()
+     # Define a symbol for the shuffle-dispatch implementation
+     # so it knows NEON is supported even though that file is
+diff --git a/blosc/shuffle.c b/blosc/shuffle.c
+index cb166ee..974f627 100644
+--- a/blosc/shuffle.c
++++ b/blosc/shuffle.c
+@@ -36,6 +36,7 @@
+ #endif  /* defined(SHUFFLE_SSE2_ENABLED) */
+ 
+ #if defined(SHUFFLE_NEON_ENABLED)
++  #include <sys/auxv.h>
```

Comment:
@mkitti this header file doesn't seem to be available on macOS

Reply:
That's true. It's a Linux specific header.

Reply:
The problem is that c-blosc2 uses symbols from that header under similar preprocessor directives below:
https://github.com/Blosc/c-blosc2/blob/main/blosc/shuffle.c#L247-L259

In particular, the following symbols are needed from that header:
1. `getauxval`
2. `AT_HWCAP`
3. `HWCAP_AMD_NEON`

We only need them if `defined(SHUFFLE_NEON_ENABLED)` and not `defined(__aarch64__)`. Maybe that's a way out for macOS?

Reply:
@giordano I worked around it in 77f9a35. It would be better to probably replace the arm neon detection system with something similar to what the vendored zlib-ng does:

https://github.com/Blosc/c-blosc2/blob/c7071132b69d976510c255b8c1a823a3516f0d54/internal-complibs/zlib-ng-2.0.3/arch/arm/armfeature.c

With these patches, however, I'm trying to stay consistent with the current logic of the code. I'll discuss more restructuring with upstream.